test/parallel/test-net-listen-multiple-calls.js

```diff
@@ -0,0 +1,17 @@
+const assert = require('assert');
+const net = require('net');
+const server = net.createServer();
+server.on('error', () => {
+  process._rawDebug('server had an error');
```

Comment:
Why not `console.error`? BTW,  if there is an error the test should fail, not just print error message